### PR TITLE
Synchronize katib and kubeflow/kubeflow

### PR DESCRIPTION
**Pull request by juliusvonkohout**
@rimolive @diegolovison 

maybe there are even more components that have been merged in the wrong order and left the master outdated.

### CODE REVIEW

File: apps/katib/upstream/components/controller/trial-templates.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ data:
         spec:
           containers:
             - name: training-container
-              image: docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v0.16.0-rc.1
+              image: docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v0.16.0
```
Comment (rimolive) [inline apps/katib/upstream/components/controller/trial-templates.yaml:36]: Looks like this image tag does not exist.

cc @andreyvelich

Comment (juliusvonkohout) [reply apps/katib/upstream/components/controller/trial-templates.yaml:36]: I am just merging what we have in 1.8.

Comment (andreyvelich) [reply apps/katib/upstream/components/controller/trial-templates.yaml:36]: Why this tag doesn't exist ?
Here it is: https://hub.docker.com/layers/kubeflowkatib/enas-cnn-cifar10-cpu/v0.16.0/images/sha256-5c27e2c36638d0764619d99c6de442ee54a8991c6bc5e0c8e5dacb9a75ddd9ed?context=repo

Comment (rimolive) [reply apps/katib/upstream/components/controller/trial-templates.yaml:36]: Hmm, I tried search for this tag in Docker Hub but can't find it. Thanks for the confirmation!

**Pull request merged**